### PR TITLE
Hoje mood recognition

### PR DESCRIPTION
**Pull request by QuincySharp**

### CODE REVIEW

File: classification/classifier/profile_data_extractor.py [hunk 1]
```diff
@@ -23,14 +23,21 @@ def get_classifier_data(data_file_name):
...
+    mood_aggressive_probability = highlevel['mood_aggressive']['probability']
+    mood_happy = highlevel['mood_happy']['value']
+    mood_happy_probability = highlevel['mood_happy']['probability']
+    mood_sad = highlevel['mood_sad']['value']
+    mood_sad_probability = highlevel['mood_sad']['probability']
 
     subprocess.run("rm {}".format(output_file_path), shell=True)
 
-    return (timbre, timbre_probability), (mood_relaxed, mood_relaxed_probability), (mood_party, mood_party_probability)
+    return (timbre, timbre_probability), (mood_relaxed, mood_relaxed_probability), (mood_party, mood_party_probability), (mood_aggressive, mood_aggressive_probability), (mood_happy, mood_happy_probability), (mood_sad, mood_sad_probability)
```
Comment (Areedtz) [inline classification/classifier/profile_data_extractor.py]: This line should probably be split up in multiple lines. It's very long.

**Comment by LukasBHolst** [comment]
Cool

File: classification/classifier/profile_data_extractor.py [hunk 2]
```diff
@@ -23,14 +23,21 @@ def get_classifier_data(data_file_name):
...
     mood_relaxed = highlevel['mood_relaxed']['value']
     mood_relaxed_probability = highlevel['mood_relaxed']['probability']
     mood_party = highlevel['mood_party']['value']
     mood_party_probability = highlevel['mood_party']['probability']
+    mood_aggressive = highlevel['mood_aggressive']['value']
+    mood_aggressive_probability = highlevel['mood_aggressive']['probability']
+    mood_happy = highlevel['mood_happy']['value']
+    mood_happy_probability = highlevel['mood_happy']['probability']
+    mood_sad = highlevel['mood_sad']['value']
+    mood_sad_probability = highlevel['mood_sad']['probability']
```
Comment (Areedtz) [inline classification/classifier/profile_data_extractor.py]: Try to add more spaces between the text to make it a tiny bit more readable.

```python
    mood_relaxed = highlevel['mood_relaxed']['value']
    mood_relaxed_probability = highlevel['mood_relaxed']['probability']

    mood_party = highlevel['mood_party']['value']
    mood_party_probability = highlevel['mood_party']['probability']

    mood_aggressive = highlevel['mood_aggressive']['value']
    mood_aggressive_probability = highlevel['mood_aggressive']['probability']

    mood_happy = highlevel['mood_happy']['value']
    mood_happy_probability = highlevel['mood_happy']['probability']

    mood_sad = highlevel['mood_sad']['value']
    mood_sad_probability = highlevel['mood_sad']['probability']
```

File: classification/__main__.py [hunk 3]
```diff
@@ -46,5 +46,6 @@ def process_data_and_extract_profiles(song_id, song_file, song_output_file):
     pool.close()
 
     print(tabulate(res,
-                   headers=['Song ID', 'Timbre', 'Mood relaxed', 'Mood party'],
+                   headers=['Song ID', 'Timbre', 'Mood relaxed', 'Mood party', 'Mood aggressive', 'Mood happy', 'Mood sad'],
```
Comment (Areedtz) [inline classification/__main__.py]: Try to make this multiple lines. An example of something that is pep8-compliant:
```python
headers=[
    'Song ID', 'Timbre', 'Mood relaxed', 'Mood party', 
    'Mood aggressive', 'Mood happy', 'Mood sad'
    ],
```

Comment (LukasBHolst) [reply classification/__main__.py]: yeah would probably make this 3 lines

File: classification/classifier/profile_data_extractor.py (hunk 1 above)
Comment (LukasBHolst) [reply classification/classifier/profile_data_extractor.py]: yeah, definetly split up

**Pull request merged**